style.css

```diff
@@ -75,4 +78,110 @@ a {
     transition: transform 0.2s;
     cursor: pointer;
     margin: 5%;
+  }
+
+.work {
+    display: flex;
+    flex-direction: row;
+    padding: 5%;
+    gap: 5%;
+}
+
+.item{
+    flex: 1;
+}
+
+.text-work1 {
+    color: #344563;
+}
+
+.text-work2 {
+    color: #6b778c;
+}
+
+.little-bottom {
+    border-radius: 6px;
+    background-color: #ebebff;
+    font-size: clamp(8px, 5vw, 12px);
+    padding-left: 1%;
+    padding-right: 1%;
+    color: #6070ff;
+}
+
+.project {
+    align-content: center;
+    padding-left: 1%;
+    padding-right: 1%;
+    border: 3px solid #6070ff;
+    color: #6070ff;
+    display: inline-block;
+    border-radius: 8px;
+    padding-top: 2px;
+}
+
+.menu {
+    height: 20px;
+    width: 20px;
+}
+
+.grid-nav {
+    list-style-type: none;
+    list-style: none;
+    display: grid;
+    align-items: center;
+    justify-content: space-between;
+    grid-template-columns: repeat(4,auto);
+}
+
+.Professional-art {
+    display: grid;
+    align-items: center;
+    grid-template-columns: 1fr 1fr;
+}
+
+.art-print {
+    display: flex;
+    flex-direction: column;
+    padding-left: 20%;
+    padding-right: 20%;
+}
+
+@media (max-width: 375px) {

```

Comment:
Awesome job with this media query for mobile styles.
I would suggest using `576px` as your breakpoint since that is an [industry standard](https://getbootstrap.com/docs/5.0/layout/breakpoints/).